schema/schema_registry.cc

```diff
@@ -164,6 +164,10 @@ schema_ptr schema_registry::get_or_load(table_schema_version v, const schema_loa
     return e.get_schema();
 }
 
+void schema_registry::flush() {
+    _entries.clear();
```

Comment:
wouldn't it be clearer to call this method clear() instead of flush()? It's not like it flushes the data to a file or anything, it really clears it.

Reply:
I can rename it.